appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/admin/MetricsServiceConfiguration.java

```diff
@@ -98,4 +98,13 @@
     String getVirtualServers();
     void setVirtualServers(String value) throws PropertyVetoException;
 
+    /**
+     * Boolean value determining if the security is enabled or not.
+     *
+     * @return
```

Comment:
I also like to keep javadoc short if there is little to say. In would in such case move the above line after the `@return` since its mostly describing the return value. Not sure if we have a guide on it.